src/test/java/com/synopsys/integration/detect/battery/detector/ConanCliBattery.java

```diff
@@ -57,11 +56,20 @@ void conanWithRevisions() {
     @Test
     void conanPkgRevOnly() {
         DetectorBatteryTestRunner test = new DetectorBatteryTestRunner("conan-pkgrevonly", "conan-cli/pkgrevonly");
-        test.executableFromResourceFiles(DetectProperties.DETECT_CONAN_PATH, "conan-info-pkgrevonly.xout");
+        setTestExecutableOutputForConan1(test, "conan-info-pkgrevonly.xout");
         test.sourceDirectoryNamed("conan-pkgrevonly");
         test.sourceFileNamed("conanfile.txt");
         test.property("detect.conan.attempt.package.revision.match", "true");
         test.expectBdioResources();
         test.run();
     }
+
+    private void setTestExecutableOutputForConan1(DetectorBatteryTestRunner test, String resource) {
```

Comment:
The main test case I see most value from is confirming the cascading is not broken in the future. Would you say this is sufficient or do we need more tests to confirm the conan1, conan2, and CLI detectors are "cascading" as they should?

Reply:
I want to add a couple of test cases for Conan 2 but thinking of doing it in a separate PR. I can also think more about cascading as I do that.